CHANGELOG.md

```diff
@@ -38,6 +38,10 @@ Read our [guidelines for writing a good changelog entry](https://github.com/biom
 
 ### Formatter
 
+#### New features
+
+- Biome now allows to format the `package.json` file. Contributed by @pattrickrice
```

Comment:
We could tell users that it is now the default behavior? And say that it can ignore using ignores config.